wowup-electron/src/app/app.component.ts

```diff
@@ -38,20 +40,13 @@ export class AppComponent implements AfterViewInit {
     private _dialog: MatDialog,
     private _addonService: AddonService
   ) {
-    this.translate.addLangs([
-      "en",
-      "de",
-      "es",
-      "fr",
-      "it",
-      "ko",
-      "nb",
-      "pt",
-      "ru",
-      "zh",
-    ]);
+    this._fileService
+      .listFiles("./src/assets/i18n/", "*.json")
```

Comment:
This path will not work when the app is published.

Reply:
rolled back to hardcoded language list